### ISSUE TITLE
Map needs a key

### ISSUE THREAD

**Issue by mvl22** [cyclestreets/cyclescape#783]
The map needs a key.

This should be a local copy of the HTML code at:
https://www.cyclestreets.net/journey/help/faq/#mapkey

**Comment by nikolai-b** [comment]
Do you mean HTML or do you just mean a link to open a new window with the image:

```html
<a 
  href="https://www.cyclestreets.net/images/general/mapkeyopencyclemap.png"
  onclick="
    window.open(
      this.href,
      'mapKey',
      'toolbar=no,location=no,status=no,menubar=no,scrollbars=no,resizable=no,width=441,height=507'
    );
    return false;
  " 
>
  Map Key
</a>
```

(with a local copy of the image)

**Comment by mvl22** [comment]
Some kind of reveal like this, preferably one which opens just below the map as a revealed panel just under the map, is needed yes.

**Comment by nikolai-b** [comment]
And do we want this on all maps using Leaflet's attribution 

![screenshot from 2018-08-05 20-26-48](https://user-images.githubusercontent.com/6289830/43689164-fa6fdd6a-98ed-11e8-9755-f153b6182b19.png)

**Comment by mvl22** [comment]
Yes, that would work well. Please avoid American Capitalisation as usual, i.e. please use "Map key".

(We already have the actual map attributions in the footer of every page.)
